File: work-dispatcher/src/main/java/works/service/WorkDispatcherMain.java
package works.service;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class WorkDispatcherMain {

    public static void main(String[] args) {
        SpringApplication.run(WorkDispatcherMain.class, args);
    }

}

File: work-dispatcher/src/main/kotlin/works/service/WorkDispatcherMain.kt
package works.service

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.stream.config.BindersHealthIndicatorAutoConfiguration

@SpringBootApplication
class WorkDispatcherMain

fun main(args: Array<String>) {
    SpringApplication.run(WorkDispatcherMain::class.java, *args)
}
